packages/pymedphys_dicom/src/pymedphys_dicom/dicom/anonymise.py

```diff
@@ -483,6 +436,15 @@ def is_anonymised_dataset(ds, ignore_private_tags=False):
             if elem.VR == "SQ":
                 if not (elem.value == [] or elem.value == dummy_value):
                     is_anonymised = False
+            elif elem.VR == "DS":
+                if not (
+                    elem.value == ""
+                    or np.isclose(float(elem.value), float(dummy_value))
+                ):
+                    is_anonymised = False
+            elif elem.VR in ("OB", "OW"):
+                if not (elem.value == "" or elem.value == (0).to_bytes(2, "little")):
+                    is_anonymised = False
```

Comment:
JAC

Can't help but think there must be a neater way to step through these cases. No real issue though. It's clear enough.

Reply:
I was thinking that too as I was writing it. I think I'll clean it up :)

Reply:
Handling a range of cases within the std lib:
https://github.com/python/cpython/blob/master/Lib/dataclasses.py#L775

Reply:
(not saying that method is appropriate here though...

Reply:
Eventually, if we just switch to a single method (so no `replace_values` parameter) that matches the recommendations [here](http://dicom.nema.org/dicom/2013/output/chtml/part15/chapter_E.html#table_E.1-1), this won't be needed anyway. Made some small changes, but might just leave it for now?

Reply:
Sure :)